app/services/taxonomy/find_childest.rb

```diff
@@ -0,0 +1,25 @@
+module Taxonomy
+  class FindChildest
```

Comment:
I think this code was more readable as `taxons_to_publish`.

Firstly, it sounds weird - childest isn't a word, and I don't think it actually describes what this does, because it could select even the root of the taxonomy if there are no descendants selected.

Maybe it's better described as "deselecting redundant taxons" or something like that.

Secondly, it's a class with a single method, and none of its attributes need to be attributes (they are just the parameters passed to the initializer). So we could accomplish the same thing with a self method.